src/extensions/extensionsHelper.ts

```diff
@@ -263,17 +269,6 @@ export function validateSpec(spec: any) {
           } cannot have options because it is type STRING`
         );
       }
-      if (
-        param.default &&
-        param.validationRegex &&
-        !RegExp(param.validationRegex).test(param.default)
-      ) {
-        errors.push(
```

Comment:
Why is this gone? Do we do regex validation elsewhere?

Reply:
This was checking that default values pass the validation_regex. However, its really hard to reliably test this for default values that reference autopopulated params, since we don't know what those are. I considered changing this to inject AUTOPOPULATED_PARAM_PLACEHOLDERS in first, and checking based on that - however, that still could let through defaults that would fail validation on some projects.

Instead of keeping an imperfect validation for this infrequent case, I figured it was better to get rid of it entirely.